### PR TITLE
Accessibility - Up to 100 (Still need to do manual tests)

### LINKED ISSUE #2: Accessibility is 38/100 for our start
Looks like there's three main parts to this (maybe make sub-issues): 
- Background and Foreground colors don't contrast enough
- html element does not have language tags
- image elements do not have alt tags

### PR DESCRIPTION
**Pull request by nicknist**
#### What's this PR do?
This PR updates the html document with alt tags, the html document with a language tag, and updates the base.scss with an adjusted color scheme for the text that was not accessible due to color contrast. 
#### Where should the reviewer start?
I would take a look at the base.scss and the index.html (both under the src folders). These are the only two files that were changed.
#### How should this be manually tested?
I would pull the repo down and then do an accessibility audit on our your machine and server. Want to make sure that this was implemented correctly and works on your computers!
#### Any background context you want to provide?
We will need to continue to check accessibility (especially tabbing as there is no tabIndex at all right now), but for now the audit comes back well. We should keep checking this and also do the manual tests.
#### What are the relevant tickets?
This should fix #2, but we still have a few accessibility issues. 
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -151,7 +152,7 @@ h3 {
 
 .hydration__article--1 span,
 .sleep__article--1 span {
-  color: var(--accent-color);
+  color: var(--accent-color2);
```
Comment (Alkiiro) [inline src/css/base.scss]: I don't have any experience with SCSS, but it looks like we assign a variable in the root/global context, and then call that variable wherever needed.  I'll pull down and ensure that accessibility is at 100%.

**Comment by khalidwilliams** [comment]
> The accessibility changes look great Nick.
> `<html lang="en">`
> Thanks for placing in the alt tags. Good descriptions for our accessibility functionality.
> `person that is tired after working out`
> Pulled down and accessibility looks to be at 100%. Thanks for updates and looks good.

@foster55f rather than making code blocks in a comment here, make the comment on the actual line of code. To do this, go to `Files Changed` at the top of the PR, then click on the loine of code you want to review itself. It helps folks view the comments within the context they refer to

**Comment by khalidwilliams** [comment]
@Alkiiro nice attachment of the screenshots

File: src/css/base.scss (hunk 1 above)
Comment (khalidwilliams) [reply src/css/base.scss]: @Alkiiro @nicknist these are actually regular CSS variables. We'll talk about Sass variables tomorrow, but try refactoring this to work with Sass variables.

Comment (khalidwilliams) [reply src/css/base.scss]: Same goes with the other instances of using variables here

File: src/index.html [hunk 2]
```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (khalidwilliams) [inline src/index.html]: Nice addition

File: src/index.html [hunk 3]
```diff
@@ -85,13 +84,13 @@ <h3 class="section__h3--title">Activity</h3>
       <section class="activity--container--1 widget--container">
         <article class="activity__article--1 article--widget">
           <h5>TODAY'S ACTIVITY</h5>
-          <img class="exercise-icon" src="./images/exercise.icon.png">
+          <img class="exercise-icon" src="./images/exercise.icon.png" alt="person running icon to signify today's activity">
```
Comment (khalidwilliams) [inline src/index.html]: Good Alt tags all around.

**Pull request merged**